src/ConcordiumNetSdk/SignKey/ISigner.cs

```diff
@@ -3,9 +3,14 @@ namespace ConcordiumNetSdk.SignKey;
 public interface ISigner
 {
     /// <summary>
-    /// Signs bytes using concrete implementation of sign key.
+    /// Gets the length of the signature produced by this signer.
     /// </summary>
-    /// <param name="bytes">the bytes to be signed.</param>
-    /// <returns><see cref="T:byte[]"/> - thw signed bytes.</returns>
+    public UInt32 GetSignatureLength();
```

Comment:
I don't believe it is, and it's not even used, since I ended up referencing `Ed25519SignKey.SignatureBytesLength` in `AccountSignatureMap`. I had other plans for this initially.